src/test/java/next/reflection/ReflectionTest.java

```diff
@@ -102,4 +105,19 @@ public void privateFieldAccess() throws NoSuchMethodException, IllegalAccessExce
 
     }
 
+    @Test
+    void createInstanceByConstructor() throws NoSuchMethodException, IllegalAccessException, InvocationTargetException, InstantiationException {
+        Class clazz = Question.class;
+
+        Constructor constructor1 = clazz.getDeclaredConstructor(String.class, String.class, String.class);
+        assertThat(constructor1.newInstance("nio", "title", "contents"))
+                .isEqualTo(new Question("nio", "title", "contents"));
+
+        Constructor constructor2 = clazz.getDeclaredConstructor(long.class, String.class, String.class, String.class, Date.class, int.class);
+        assertThat(constructor2.newInstance(1L, "nio", "title", "contents", new Date(), 0))
```

Comment:
```suggestion
        assertThat(constructor2.newInstance(1L, "nio", "title", "contents", new Date(), 1))
```
이렇게 변경해도 테스트가 통과합니다. 원인을 파악해주시고 `Question` 클래스의 수정없이 검증을 해보시겠어요?

Reply:
테스트가 통과하는건 questionId 가 동일하면 equals 로 판단하고 있어서 다른 프로퍼티들과 관계 없이 통과합니다. 😅 

생성자가 정상적으로 호출되었는지를 테스트하기에 인스턴스 비교는 부적절 하겠네요. 프로퍼티를 체크하는 형태로 수정하도록 하겠습니다.